components/magnolia/components/Navigation.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  BreadcrumbLink,
+  Button,
+  Flex,
+  Menu,
+  MenuButton,
+  MenuItem,
+  MenuList,
+} from '@chakra-ui/react';
+import { languages } from '~/utils';
```

Comment:
languages exist in @components/magnolia/api, use getLanguages() lets keep every thing together until we figure out what our core library might look like. 
I'm not a big fan of util files LOL ;)

Reply:
done! 😊